examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>may be rephrase as "We can take a look what required code changes are automatically applied by Merlin Models:"</p><p> </p><p>who is doing  scaling the learning rate by number of workers? where and how it is happening? if this is happening under the hood can you please refer to the code base? </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>It is mentioned that <code>hvd.init()</code> is necessary, but could not find it in the script below.</p>